streaming/base/converters/__init__.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Base module for downloading/uploading files from/to cloud storage."""
```

Comment:
cc: @karan6181

Do we have strong opinions about where we draw the lines between `scripts/` and `streaming/base/converters/`?

I have been pining for a `streaming/cli/` directory containing our collection of tools to operate on streaming datasets, which would have included downloading/uploading to remote...

Reply:
Any comments? @karan6181